srcpkgs/batmon/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'batmon'
+pkgname=batmon
+version=0.8
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="gnustep-make gcc-objc"
+makedepends="gnustep-base-devel gnustep-gui-devel"
+depends="gnustep-base gnustep-gui gnustep-back"
```

Comment:
These `depends` should be detected automatically.

Reply:
`gnustep-base` and `gnustep-gui` will, but `gnustep-back` won't.